languagetool-language-modules/ga/src/main/resources/org/languagetool/resource/ga/irish_tags.txt

```diff
@@ -0,0 +1,3717 @@
+Abr
+Abr:Title
+Adj:Base
+Adj:Base:DeNom
+Adj:Base:DeNom:Com:Sg
+Adj:Base:DeNom:DefArt
+Adj:Base:DeNom:Ecl
+Adj:Base:DeNom:Gen:Pl:Ecl
+Adj:Base:DeNom:Gen:Sg:Len
+Adj:Base:DeNom:Len
+Adj:Base:Ecl
+Adj:Base:hPref
+Adj:Base:Len
+Adj:Com:NotSlen:Pl
+Adj:Com:Pl
+Adj:Com:Slender:Pl
+Adj:Com:Slender:Pl:Len
+Adj:Comp
+Adj:Comp:Len
+Adj:Fem:Com:Sg
+Adj:Fem:Com:Sg:Len
+Adj:Fem:Gen:Sg
+Adj:Fem:Voc:Sg
+Adj:Fem:Voc:Sg:Len
+Adj:Gen:Strong:Pl
+Adj:Gen:Weak:Pl
+Adj:Its
+Adj:Masc:Com:Pl
+Adj:Masc:Com:Sg
+Adj:Masc:Com:Sg:Len
+Adj:Masc:Gen:Sg
+Adj:Masc:Gen:Sg:Len
+Adj:Masc:Voc:Sg
+Adj:Masc:Voc:Sg:Len
+Adj:Voc:Pl
+Adv:Dir
+Adv:Dir:Len
+Adv:Gn
+Adv:Its
+Adv:Loc
+Adv:Q
+Adv:Q:Past
+Adv:Temp
+Adv:Temp:Gen:Sg
+Art:Gen:Sg:Def:Fem
+Art:Pl:Def
+Art:Sg:Def
+CC:Cop:Pro:Q
+CC:Noun:Masc:Com:Pl
```

Comment:
I have to maintain the upstream analyser this dictionary is derived from, and it really makes no sense for me to go back and forth between two different tagsets. 
Also, single character tagging schemes are appallingly unreadable :)